integration_tests/src/main/python/marks.py

```diff
@@ -16,6 +16,7 @@
 
 allow_non_gpu_databricks = pytest.mark.allow_non_gpu_databricks
 allow_non_gpu = pytest.mark.allow_non_gpu
+ansi_mode_disabled = pytest.mark.ansi_mode_disabled
```

Comment:
Now that I look at this, it would be better to use an imperative/active voice for directives:
```suggestion
disable_ansi_mode = pytest.mark.disable_ansi_mode
```
Notice that all the other annotations are in active voice.

Reply:
P.S. Apologies for the churn.